### ISSUE TITLE
Background tcp server / udp rx in connected standby

### ISSUE THREAD

**Issue by karthikveeramani** [Microsoft/Windows-universal-samples#961]
I'm trying to run a tcp server or udp receiver in a background uwp task that's available in connected standby. I started with SocketActivityStreamSocket and verified that, I can run the CLIENT on a connected standby PC in the background, and the server on another PC. This works. Now I flipped it around to run the server on the standby PC. I get an exception ("The attempted operation is not supported for the type of object referenced") when BindServiceNameAsync or BindEndpointAsync is called while EnableTransferOwnership is configured with SocketActivityConnectedStandbyAction.Wake.

Things seem to work while using DoNotWake option, but that's not what I want.

I came across several threads that reported similar issues but didn't have closure:
https://social.msdn.microsoft.com/Forums/sqlserver/en-US/1f2e3080-5e92-4a46-bb07-6f04b1630a02/network-communications-in-the-background?forum=wpdevelop
https://github.com/anhoavu/BackgroundServer  (Problem section)

My changes look very similar to the first link. Roughly,

```
var builder = new BackgroundTaskBuilder();
builder.Name = TaskNameSocketActivity;
builder.TaskEntryPoint = "BackgroundTasks.SocketActivity";
builder.SetTrigger(new SocketActivityTrigger());
BackgroundTaskRegistration _task = builder.Register();

_tcpListener = new StreamSocketListener();
_tcpListener.EnableTransferOwnership(_task.TaskId, SocketActivityConnectedStandbyAction.Wake);
_tcpListener.ConnectionReceived += _tcpListener_ConnectionReceived;
//await _tcpListener.BindServiceNameAsync("80"); => fails saying that I need to give a specific IP while using with Wake option
await _tcpListener.BindEndpointAsync(new HostName(myIP), "40404");  // Fails with exception said above

```
So in summary I have tried the following combinations in the background with Wake option:
1. TCP server and UDP receiver
2. Ethernet and WiFi  (to check if it could be a driver issue with an adapter)
3. BindServiceNameAsync (this seems not supported by design) and BindEndpointAsync (this should have worked but didn't)
None of these combinations worked.
 
Can someone from Microsoft help?

**Comment by oldnewthing** [comment]
This is not an issue with a sample in this repo. It appears to be an issue with your own app. You can open a support case with our Developer Support team:

1. Visit http://aka.ms/storesupport and sign-in with your Microsoft account
2. Under the “App Development” section choose "Windows 10 universal app development", then choose the appropriate problem type and category to get engaged through a formal support channel.
